lib/autoload/sfCoreAutoload.class.php

```diff
@@ -9,7 +9,7 @@
  */
 
 // The current symfony version.
-define('SYMFONY_VERSION', '1.6.2');
+define('SYMFONY_VERSION', '1.6.4');
```

Comment:
`1.6.3` was used by @marcinkantarowski's recent release - we forgot to bump the version for that at the time.